README.md

```diff
@@ -58,44 +84,25 @@ secrets:
         external: true
 ```
 
+## Development
+
 ### Running Locally
 
 The tool has been designed specifically to run using docker swarm or compose but it can be run locally if required. I have tryed my best to remove all issues when running locally but it may not be perfect.
 
-* First Clone the repository using following the command:
+1. Clone the repository using following the command:
 ```
 https://github.com/uccser/swarm-stack-updater.git
 ```
-* Then use the following command to pull BATS, which is used for automated testing:
+2. Use the following command to pull BATS, which is used for automated testing. This will pull all the required tools for running and creating tests for this application.
 ```
 git submodule update --init
 ```
-This will pull all the required tools for running and creating tests for this application.
-
-
-
-## Requirements
-* The tool makes use of the github api and container registry. You may have to update the script to work for other 
-repositories and container registries.
-* Website must have a status url that returns a JSON object that contains both 
-sha commit hash and a tag version.
-* ```{"VERSION_NUMBER": "3.10.0", "GIT_SHA": "a49a111d"}```
-* Deployment on swarm requires swarm cronjob running and deployed on swarm
-this can be found here https://github.com/crazy-max/swarm-cronjob
-* From the bottom of the compose file you can see that a secret is required. This should be a github_access_token to allow for multiple requests. More about
-docker secrets can be found here: https://docs.docker.com/engine/swarm/secrets/
-* Your docker images should also contain some extra information in their manifest files. This is so the app is able to identify whether a new image is available to download.
+3. Ensure that you have a configuration file created and have the requried environment varibles defined. To define environment varibles use the command ```export ENV_VAR=<value>```.
 
-Manifest JSON should contain at least:
-```
-...
-labels: {
-  "org.opencontainers.image.revision": "<commit-sha>"
-}
-...
-```
+4. Run the swarm updater by running the command ```./src/swarm_stack_updater.sh.``` You may need to set to to be executible this can be done by typing the command ```chmod +x /src/swarm_stack_updater.sh```
```

Comment:
```suggestion
4. Run the swarm updater by running the command ```./src/swarm_stack_updater.sh```.
  You may need to set to to be executible this can be done by typing the command ```chmod +x /src/swarm_stack_updater.sh```
```

Full stop was inside code block.